nextest-runner/src/platform.rs

```diff
@@ -21,22 +23,52 @@ pub struct BuildPlatforms {
     /// In the future, this will become a list of target triples once multiple `--target` arguments
     /// are supported.
     pub target: Option<TargetTriple>,
+
+    /// The sysroot for the target platform.
+    ///
+    /// This is needed to run procedure macro binaries on Windows. See
+    /// [https://github.com/nextest-rs/nextest/issues/1493] for details. Empty if failed to detect.
```

Comment:
Note that this is actually required on all platforms per #267, it's just that on other platforms rustup still sets the library path env vars. Could you update the comment to reflect that?

Reply:
Done.